inginious/frontend/static/js/admin.js

```diff
@@ -0,0 +1,77 @@
+//
+// This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+// more information about the licensing of this file.
+//
+function revoke_binding(event){
+    var username = $(this).data("username");
+    var binding_id = $(this).data("binding");
+    $.post({
+      url: "user_action",
+      data: {
+        "username":username,
+        "action":"revoke_binding",
+        "binding_id": binding_id
+      },
+    }).done(function(feedback) {
+        if (!feedback.hasOwnProperty('error')){
+            window.location.href = "users";
```

Comment:
Do you actually need this redirection ? Is this working without any prefix (hostname and path prefix) required ?

Reply:
I used this to close (and refresh page) the modal after hitting submit button. If i remove this, the page just stay on the open modal. Yeah working without prefix :)

Reply:
If you want to close the modal and refresh the page, maybe you should set the button ``type`` attribute to ``submit``.

Reply:
As we said in private, it displays a feedback in the modal so I cannot use the `type="submit"` solution. I mark this as resolved for now.